posthog/models/plugin.py

```diff
@@ -252,8 +252,14 @@ class Meta:
     # Used in the frontend
     name: models.CharField = models.CharField(max_length=400, null=True, blank=True)
     description: models.CharField = models.CharField(max_length=1000, null=True, blank=True)
-    # Used in the frontend to hide pluginConfgis that user deleted
+    # Used in the frontend to hide pluginConfigs that user deleted
     deleted: models.BooleanField = models.BooleanField(default=False, null=True)
+    match_action = models.ForeignKey(
```

Comment:
could you add a comment above this what this field is/does

optional: we could have multiple actions here via [ManyToMany](https://docs.djangoproject.com/en/5.0/topics/db/examples/many_to_many/), though we'd need to update https://github.com/PostHog/posthog/blob/master/plugin-server/src/utils/db/sql.ts#L94C23-L94C42 & `runComposeWebhook` accordingly too. We can initially still limit it to be a single action in the UI. I'm not sure if you're thinking longer term you'll want multiple actions/events to be able to be picked or not 🤷‍♀️  it's not that different here at this point to make it work over multiple.

Reply:
I don' think many to many will make much sense tbh. Makes it simpler this way, and you can solve many to many by multiple match groups in the action.